src/Components/Signin/index.js

```diff
@@ -12,17 +12,13 @@ import {
   FormLabel,
   FormWrap,
   Text,
-
 } from "./SigninElements";
 import { useState } from "react";
-import { forgotPassword } from "../Forgot_password/Forgot_password";
+
 import { NavLink } from "react-router-dom";
```

Comment:
```suggestion

import { NavLink } from "react-router-dom";
```

remove extra line for code format